content/docs/concepts/multicluster-deployments/index.md

```diff
@@ -12,39 +12,94 @@ a single controllable composite application.
 
 For applications up to a certain size, all of the microservices comprising the
 application can be running on a single orchestration platform (e.g., Kubernetes cluster).
-However, for many reasons such as scale, redundancy, compliance, etc., most applications
-will eventually need to be distributed and have some of their services running elsewhere.
+Eventually, you may need to distribute most applications and run some or all of their services elsewhere due to one or more of the following reasons:
+
+- Scale
+- Redundancy
+- Workload isolation
+- Compliance
+- Integration with legacy services
 
 Istio supports many possible topologies for distributing the services of an
 application beyond a single cluster, for example:
 
-* Services within the mesh can use [service entries](/docs/concepts/traffic-management/#service-entries)
+- You can use [egress traffic control](/docs/tasks/traffic-management/egress/) to allow workloads inside the mesh
   to access standalone external services or to access services exposed by another loosely-coupled service mesh.
-* You can [expand the service mesh](/docs/setup/kubernetes/additional-setup/mesh-expansion/) to include
+- You can [expand the service mesh](/docs/setup/kubernetes/additional-setup/mesh-expansion/) to include
   services running on VMs or bare metal hosts.
-* You can combine the services from more than one cluster into a single composite service mesh.
+- You can combine the services from more than one cluster into a single composite service mesh.
+- You can combine autonomous service meshes in multiple clusters into a loosely coupled [federation of service meshes](#mesh-federation).
+
+## Single multicluster service mesh or mesh federation?
+
+As you plan your multicluster topology, you need to determine if you should combine the clusters into
+[a single multicluster mesh](#multicluster-service-mesh) or create [a federation of multiple meshes](#mesh-federation).
+To make that decision, ask yourself the following questions:
+
+1. Do you enforce **{{< gloss >}}uniform naming{{< /gloss >}}** in your clusters?
+
+1. Do you want your combined clusters to **expose** all their services to each other? Due to compliance or security
+   reasons, you might need {{< gloss >}}selective service exposure{{< /gloss >}}.
+
+1. Are you required to **control all the traffic that enters your cluster**?
```

Comment:
Can we clarify this? I'm not entirely sure what control means here, maybe examples or some more specific wording?

Reply:
This is a defense-in-depth/compliance requirement: you have to put a "firewall" at the entrance to your cluster and explicitly control who can call specific services in your cluster. Ingress gateway can function as such a firewall, in my opinion. This is in addition to the RBAC policies inside the cluster. In the current, single mesh implementations, the passthrough gateways allow all the traffic in. This cannot satisfy, for example, PCI DSS compliance, which requires network segmentation + a firewall between Cardholder Data Environment and other environments, in particular between Cardholder Data Environment and DMZ network.

Reply:
OK, I think I'm a bit thrown off by the word cluster.

In my model if we think about groups of clusters that need to be isolated, each of those are a mesh, so this is about controlling all traffic that enters some subset of clusters, not necessarily each cluster. Maybe we can reword to be about groups and use an example of requiring firewalling between environments, and each environment should be its own mesh.